### ISSUE TITLE
Training VQVAE dose not convergence.

### ISSUE THREAD

**Issue by henanjun** [AntixK/PyTorch-VAE#71]
- This is the reconstruction result  of 100 Epoch of VQVAE.  
![recons_VQVAE_Epoch_99](https://user-images.githubusercontent.com/27271981/204431756-8dd9346a-5ed3-4560-951c-558f1f31e455.png)

**Comment by blade-prayer** [comment]
I found that in vq_vae.yaml, the *scheduler_gamma* is set to be 0.0. This parameter controls the multiplicative factor in *torch.optim.lr_scheduler.ExponentialLR*, which makes the learning rate to be 0 after epoch 0. Do you think this is the reason?

**Comment by imskull** [comment]
Changing "LR"(learning rate" from 0.005 to 0.001 helps.

**Comment by xjtupanda** [comment]
> Changing "LR"(learning rate" from 0.005 to 0.001 helps.

Met the same problem. I found the loss always unreasonably high (~1.0e+6) and it might cause a gradient explosion. This one helps, thanks a lot.

**Comment by ohhh-yang** [comment]
In my training process the loss is more unreasonable(up to 1.0e+26!). And it just fluctuates like a pendulum. This one really works, you are my god!!!

> Changing "LR"(learning rate" from 0.005 to 0.001 helps.

**Comment by Yysrc** [comment]
> Changing "LR"(learning rate" from 0.005 to 0.001 helps.

It really helps, thanks a lot!

**Comment by unistdJRZ** [comment]
vqvae's codebook may collaps easily
